src/test/functional/steps/court-types.ts

```diff
@@ -53,17 +50,67 @@ When('I uncheck a court type', async () => {
 });
 
 Then('a court types error message is displayed', async () => {
-  const elementExist = await I.checkElement('#courtTypesTab .govuk-error-summary');
+  const elementExist = await I.checkElement('#courtTypesContent > div.govuk-error-summary > div > ul > li');
   expect(elementExist).equal(true);
-
 });
 
 When('I check a court type which has code associated with it', async () => {
   const selector = '#court_types';
   const elementExist = await I.checkElement('#court_types');
   expect(elementExist).equal(true);
-
   await I.click(selector);
+});
+
+Then('I will make sure that one of the court type is selected', async () => {
```

Comment:
maybe change this to pass the selector instead? I.e 

Then('I assert that the court type of {} is selected', async (selector: string) => {